src/main/java/org/mifos/connector/ams/paygops/camel/route/PaygopsRouteBuilder.java

```diff
@@ -66,20 +72,30 @@ public void configure() {
                 .log(LoggingLevel.INFO, "Paygops Validation Response Received")
                 .process(exchange -> {
                     // processing success case
-                    String body = exchange.getIn().getBody(String.class);
-                    ObjectMapper mapper = new ObjectMapper();
-                    PaygopsResponseDto result = mapper.readValue(body, PaygopsResponseDto.class);
-                    logger.info("body : "+ result);
-                    //JSONObject jsonObject = new JSONObject(body);
-                    if (result.getReconciled()){
+                    try {
+                        String body = exchange.getIn().getBody(String.class);
+                        logger.info("raw reply : " + body);
```

Comment:
log level for this should not be info. for printing request response, recommended log level is trace